app/src/androidTest/java/com/github/sdp_begreen/begreen/firebase/FirebaseDBTest.kt

```diff
@@ -253,4 +254,200 @@ class FirebaseDBTest {
         }
     }
 
+    @Test
```

Comment:
All these "followers" tests should be in a separate class from `FirebaseDBTest` IMO. Maybe make FirebaseDBTest an abstract class if there are common utility functions and configuration functions you want to reuse.

Reply:
We'll try to do that, thanks !